solvers/aeossp_standard/mwis_conflict_graph/src/solve.py

```
@@ -0,0 +1,299 @@
+                candidate_config=candidate_config,
+                candidate_summary=candidate_summary,
+                graph=graph,
+                mwis_config=mwis_config,
+                mwis_stats=mwis_stats,
+                repair_config=repair_config,
+                repair_result=repair_result,
+                timing_seconds=timing_seconds,
+                candidates=candidates,
+                selected_candidates=selected_candidates,
+            )
+    except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)